gadget/install/install.go

```diff
@@ -166,11 +179,17 @@ func Run(model gadget.Model, gadgetRoot, kernelRoot, device string, options Opti
 		// matches what is on the disk, but sometimes there may not be a sector
 		// size specified in the gadget.yaml, but we will always have the sector
 		// size from the physical disk device
-		if err := makeFilesystem(&part, diskLayout.SectorSize); err != nil {
+		timings.Run(perfTimings, fmt.Sprintf("make-filesystem[%s]", part.Role), fmt.Sprintf("Create filesystem for %s", part.Role), func(timings.Measurer) {
```

Comment:
If the gadget has some non-ubuntu partitions, the role will likely be unset for those. Maybe we should use a label in this case?